packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import { ArrowRight20, ArrowDown20, Pdf20 } from '@carbon/icons-react';
+import '../../../../../styles/scss/components/buttongroup/_buttongroup.scss';
+import readme from '../README.md';
+import { BUTTON_GROUP } from '../../../internal/FeatureFlags';
+
+import ButtonGroup from '../ButtonGroup';
+
+if (BUTTON_GROUP) {
+  storiesOf('ButtonGroup', module)
+    .addDecorator(withKnobs)
+    .addParameters({
+      readme: {
+        sidebar: readme,
+      },
+    })
+    .add('Default', () => {
+      const iconMap = {
+        ArrowRight20,
+        ArrowDown20,
+        Pdf20,
+      };
+
+      const iconOptions = {
+        None: null,
+        'Arrow Right': 'ArrowRight20',
+        'Arrow Down': 'ArrowDown20',
+        PDF: 'Pdf20',
+      };
+
+      const primaryIcon =
+        iconMap[
+          select('Primary button icon', iconOptions, iconOptions['Arrow Right'])
+        ];
+      const secondaryIcon =
+        iconMap[
+          select('Secondary button icon', iconOptions, iconOptions['PDF'])
+        ];
+
+      const buttons = [
+        {
+          link: '',
+          copy: text('Primary button copy', 'Primary action button'),
+          renderIcon: primaryIcon || null,
+        },
+        {
+          link: '',
+          copy: text('Secondary button copy', 'Secondary action button'),
+          renderIcon: secondaryIcon || null,
+        },
+      ];
```

Comment:
@jeffchew don't kill me for suggesting this, but it might be good if we could provide an option to add or remove buttons.

```javascript
const buttonCount = number('Number of buttons', 2);
const buttons = [];

for (let i = 0; i < buttonCount; i++) {
  buttons.push({
    link: text(`Link ${i+1}`, `https://example.com`),
    copy: text(`Button ${i+1}`, `Button ${i+1}`),
    renderIcon: iconMap[select(`Icon ${i+1}`, iconOptions, iconOptions['Arrow Right'])],
  })
}
```

Reply:
@wonilsuhibm @larahanlon2 @oliviaflory what say you? 😄

Reply:
@photodow regardless of the decision, I don't think this is a deal breaker for this PR, especially with the fact this is still behind a feature flag and should suffice for your Modal story. We can decide this later perhaps.

Reply:
Issue to track this: https://github.com/carbon-design-system/ibm-dotcom-library/issues/558